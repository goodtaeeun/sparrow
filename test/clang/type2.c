enum a { b, c };
typedef f;
struct f {
  enum a d;
};
struct f a = {c};
int main() {
  f x = a.d;
  return x;
}
