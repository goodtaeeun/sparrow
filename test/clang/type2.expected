/* Generated by CIL v. 1.7.4 */
/* print_CIL_Input is false */

#line 1 "type2.c"
enum a { b = 0, c = 1 };
#line 2 "type2.c"
typedef int f;
#line 3 "type2.c"
struct f {
  enum a d;
};
#line 6 "type2.c"
struct f a = {(enum a)1};
#line 7 "type2.c"
int main(void) {
  f x;

  {
#line 8
    x = (f)a.d;
#line 9
    return (x);
  }
}
